notifier.go

```diff
@@ -337,7 +337,11 @@ func (n *Notifier) NotifyOnPanic() {
 	if v := recover(); v != nil {
 		notice := n.Notice(v, nil, 2)
 		notice.Context["severity"] = "critical"
-		_, _ = n.SendNotice(notice)
+		_, err := n.SendNotice(notice)
+		if err != nil {
+			logger.Printf("Unable to send notice: %v", err)
```

Comment:
There is already some logging in https://github.com/airbrake/gobrake/blob/c7d7be486493c97e046abf8045c886e25c3a99f0/notifier.go#L302-L303. I guess it is not enough, but duplicating it here is not a solution too...